cpp/unreal_projects/SpearSim/Python/validate_scene_kujiale.py

```diff
@@ -0,0 +1,591 @@
+import argparse
+import numpy as np
+import pathlib
+import posixpath
+import unreal
+import spear
+
+
+def validate_level(level):
+    #
+    # TODO:
+    # We name each scene using a lower_case_with_underscore naming convention and a four
+    # digit suffix (e.g., "apartment_0000", "apartment_0001", etc).
+    #
+
+    pass
+
+
+def validate_content_browser(level):
+    #
+    # TODO:
+    # There should be exactly one umap file with the same name as the scene in the scene's
+    # Maps directory. Our code imposes this requirement to support loading scenes by name.
+    #
+
+    #
+    # TODO:
+    # The "Content Browser" should not contain unreferenced assets, except for in the
+    # "Debug" directory.
+    #
+
+    pass
+
+
+def validate_outliner(level, iterate_all_actors):
+    editor_actor_subsystem = unreal.get_editor_subsystem(unreal.EditorActorSubsystem)
+
+    actors = editor_actor_subsystem.get_selected_level_actors()
+    if len(actors) == 0 and iterate_all_actors:
+        actors = editor_actor_subsystem.get_all_level_actors()
+
+    actors = sorted(actors, key=(lambda actor: get_debug_string_actor(actor)))
+    print(len(actors))
+    for actor in actors:
+        validate_actor(level, actor)
+
+
+def validate_actor(level, actor):
+    #
```

Comment:
Add a line of whitespace after the function signature here to match our existing convention.